CMSgov/mdct-mcr
# add banner API methods and related baseline setup (MDCT-284 step 1)

## Description
We're gonna need to store banners in the database. This PR is the first step toward completion of the ticket below.

This PR sets some good baseline patterns for us to build on with other DB stuff in the future. Just another step toward completion of the ticket below.

_Note: This PR is way too big. Please forgive our mess while we get a solid baseline set. 👷‍♂️🏗️👷‍♂️_

Related ticket: [MDCT-284](https://qmacbis.atlassian.net/browse/MDCT-284)

### How to test
1. Make sure you change your `.env` files to match the updated ones in the readme.
2. Run and navigate to /admin.
3. Can click each of 'write', 'get' and 'delete' buttons to perform operations on an arbitrary mock banner.
4. All 200s 🎉 🤞 

### Changed Dependencies
Bumped `cmsgov/design-system` to `^3.4.0` 'cause they just fixed some DateField stuff we needed.

## Code author checklist

## Reviewer checklist (two different people)
